CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Changes in the upcoming versions.
 
 ## Improvements
 
-# release-2024-09-03
+# release-2024-09-10
```

Comment:
Shouldn't there be some changes?

Reply:
Oh, yes. I prepared the release last week without releasing. But we did some changes that should go here, yes.